tests/test_olio_grid_functions.py

```diff
@@ -0,0 +1,236 @@
+import os
+import numpy as np
+import pytest
+
+from resqpy.olio.grid_functions import left_right_foursome, infill_block_geometry, resequence_nexus_corp, random_cell, determine_corp_ijk_handedness, determine_corp_extent, translate_corp, triangles_for_cell_faces, actual_pillar_shape
+import resqpy.model as rq
+import resqpy.grid as grr
+
+
+def test_infill_block_geometry():
+
+    # --------- Arrange ----------
+    x_ = np.array([2, 4, 6, 8])
+    y_ = np.array([1, 3, 5, 7])
+    depth_ = np.array([10.0, 30.0, 50.0, 70.0])
+    x, y, depth = np.meshgrid(x_, y_, depth_)
+    extent = (4, 4, 4)
+    thickness = np.ones(extent) * 15
```

Comment:
Personally I prefer to use `thickness = np.full(extent, 15)` when setting a numpy array to a constant value.